backend/app/core/config.py

```diff
@@ -13,10 +14,40 @@ class ServerSessionsSettings(BaseSettings):
     expires_in: int
 
 
+class MatrixBotSettings(BaseSettings):
+    homeserver: str
+    user_id: str
```

Comment:
We don't actually need user_id as we can use /whoami with the access token to ask the homeserver who the user is.